packages/elements/src/password-field/index.ts

```diff
@@ -59,6 +59,33 @@ export class PasswordField extends TextField {
   @state()
   private isPasswordVisible = false;
 
+  @state()
+  private liveRegionContent: TranslateDirectiveResult = '';
```

Comment:
It would be better if change this to getter, please see the ef-pagination element.

Reply:
sticking with the current implementation as properties are more performant in rendering.